test/lib/rubycritic/browser_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
+require "rubycritic/browser"
+require "rubycritic/platforms/linux"
+require "rubycritic/platforms/darwin"
+require "rubycritic/generators/html_report"
+
+describe Rubycritic::Browser do
+  before do
+    @report_path = "tmp/rubycritic/overview.html"
+    @browser = Rubycritic::Browser.new @report_path
+  end
+
+  describe "open report on diffrent platform" do
+    context "on linux" do
+      it "open with default browser" do
+        Rubycritic::Config.set
+        Rubycritic::Browser.any_instance.stubs(:platform).returns(Rubycritic::Platforms::Linux.new)
+        @linux = @browser.platform
+        @linux.browser.must_equal "firefox"
+      end
+
+      it "open with google chrome browser " do
+        Rubycritic::Config.stubs(:open_with).returns(:chrome)
+        Rubycritic::Browser.any_instance.stubs(:platform).returns(Rubycritic::Platforms::Linux.new)
+        @linux = @browser.platform
+        @linux.browser.must_equal "google-chrome"
+      end
+
+      it "open with chromium browser " do
+        Rubycritic::Config.stubs(:open_with).returns(:chromium)
+        Rubycritic::Browser.any_instance.stubs(:platform).returns(Rubycritic::Platforms::Linux.new)
+        @linux = @browser.platform
+        @linux.browser.must_equal "chromium-browser"
+      end
+    end
```

Comment:
Sure, push those changes and fix the typo on "diffrent" and it'll be good to go :)